keyboards/planck/keymaps/pickle_jr/config.h

```diff
@@ -0,0 +1,40 @@
+#ifndef CONFIG_USER_H
```

Comment:
The first four lines (up to `#include "config_common.h"`) could be replaced with `#pragma once`, and the `#endif` at the end removed.